packages/infrastructure/kube_pg_cluster/main.tf

```diff
@@ -367,12 +369,12 @@ resource "kubernetes_manifest" "postgres_cluster" {
             shared_preload_libraries   = ""
             ssl_max_protocol_version   = "TLSv1.3"
             ssl_min_protocol_version   = "TLSv1.3"
-            wal_keep_size              = "2GB"
+            wal_keep_size              = var.pg_wal_keep_size_gb
             wal_level                  = "logical"
             wal_log_hints              = "on"
             wal_receiver_timeout       = "5s"
             wal_sender_timeout         = "5s"
-            max_slot_wal_keep_size     = "10GB"
+            max_slot_wal_keep_size     = "${var.pg_wal_keep_size_gb * 3}GB"
```

Comment:
@fullykubed previously this was a 5x ratio, let me know if it should be 5x or if 3x is good enough. In implentio's previous configuration, it was 1x.

Reply:
Its not really about being a multiplier. Its about setting a cap on the WAL size so disk space isn't exhausted. It is set to 10GB to not exceed the minimum storage size.

If you are going to make it a multiplier, then you need to add this value to all of the other storage parameters and limits.